contrib/forms/example2/style.css

```diff
@@ -3,72 +3,72 @@
  */ 
 
 h1 {
-    font-size: 120%;
-    font-weight: bold;
+  font-size: 120%;
+  font-weight: bold;
 }
 
 #form_container {
-    border: 1px solid black;
-    background-color: #ffffcc;
```

Comment:
Is is supposed to change the colors from `#ffffcc` to `#ffc` ?

Reply:
I think is shorthand to represent the `#ffffcc`

Reply:
yep, I think you are right. Since I noticed below, some colors were truncated and others not. So guessing the basic colors have shorthands.

Reply:
@bradymiller Yep all the colors get converted to the shorthand versions.